.github/workflows/smoketest-android.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
A question.  Instead of creating a second file that needs to be kept in sync, would it be better to just add `if` conditions to the steps we don't want to run on pull request?

Reply:
I am unsure if that is easier or harder to maintain. If conditions would make a single file so it would not duplicate bootstrap/checkout/onebuild/uploadsize

however it would add an if in every single place and I believe we duplicate bootstrap/checkout/uploadsize across all yaml files anyway. The benefit for separate files to me is clearer changes and naming (jobs will be named smoke, file edits would be obvious).

If you think that `if` is a much better approach I would do so. If it is a wash, then I would prefer to go lazy and keep the changes as is. I think for matrices I want to split as well because single CI restarts in a matrix restarts too much.

Reply:
I don't have super-strong views here.  Just worried a bit about smoketests and full test getting out of sync....

Reply:
I will take the 'no strong views' then as can keep it as is.

I think chances of out of sync are somewhat low: we only seem to ever add and the rest of the copied things are copied everywhere (bootstrap and such) so we do not care as much.